ls1intum/Artemis
# change the style of the lock information in the complaints view to av…

### Checklist

### Motivation and Context


resolve #1383 
### Description

change the color of the Lock information to black and change the Information from "Locked by..." to "original assessor is..." to avoid confusion
### Steps for Testing


1.Click on evaluate complaint and check that the lock information is present and avoids misunderstandings


### Screenshots


![image](https://user-images.githubusercontent.com/50635005/82238164-23bc1080-9937-11ea-875c-05faf4f6c9ca.png)
